public_app/pages/MoviesDetails.js

```diff
@@ -0,0 +1,114 @@
+import React    from 'react';
+import BigImg   from '../components/BigImg';
+import Nav      from '../components/Nav';
+import Details  from '../components/Details';
+import LogOut   from '../components/LogOut';
+import axios    from 'axios';
```

Comment:
Global libraries go first and then goes your imports in the list, so react and axios would be first